### ISSUE TITLE
Cannot find module '####' (undefined)​​ wallaby-webpack.js

### ISSUE THREAD

**Issue by jkyoutsey** [wallabyjs/public#1709]
### Issue description or question
Wallaby Tests Output tab gives me this output despite the fact that I have over 1100 tests. And if I run `ng test` I'll get all 1100+ tests passing.  There is no indication of where/what is failing.

```
​​​​​0 failing tests, 946 passing​​​​​
​​​​​​​​​​
​​Uncaught Error: Cannot find module '1922' (undefined)​​
​​at http://localhost:50829/wallaby-webpack.js?1528903296577:1​​
```
And this appears in the Wallaby Console Output
```
​​​​​[Info]​​​​​ Wed, 13 Jun 2018 15:50:44 GMT wallaby:workers Sandbox (active) [5jjkn] error: Uncaught Error: Cannot find module '1681' (undefined)
​​[Error] Runtime error: Uncaught Error: Cannot find module '1681' (undefined)​​
​​[Error] at http://localhost:57653/wallaby-webpack.js?1528905041230:1​​
​​​​​[Info]​​​​​ Wed, 13 Jun 2018 15:50:44 GMT wallaby:workers Failed to map the stack to user code, entry message: Uncaught Error: Cannot find module '1681' (undefined), stack: Uncaught Error: Cannot find module '1681' (undefined)
​​​​​[Info]​​​​​ at http://localhost:57653/wallaby-webpack.js?1528905041230:1
```

### Wallaby.js configuration file

```javascript
var wallabyWebpack = require('wallaby-webpack');
var path = require('path');

var compilerOptions = Object.assign(
  require('./tsconfig.json').compilerOptions,
  require('./src/tsconfig.spec.json').compilerOptions);

compilerOptions.module = 'CommonJs';

module.exports = function (wallaby) {

  var webpackPostprocessor = wallabyWebpack({
    emitModulePaths: true,

    entryPatterns: [
      'src/**/wallabyTest.js',
      'src/**/*spec.js'
    ],

    module: {
      rules: [
        { test: /\.css$/, loader: ['raw-loader', 'css-loader'] },
        { test: /\.html$/, loader: 'raw-loader' },
        { test: /\.ts$/, loader: '@ngtools/webpack', include: /node_modules/, query: { tsConfigPath: 'tsconfig.json' } },
        { test: /\.js$/, loader: 'angular2-template-loader', exclude: /node_modules/ },
        { test: /\.json$/, loader: 'json-loader' },
        { test: /\.styl$/, loaders: ['raw-loader', 'stylus-loader'] },
        { test: /\.less$/, loaders: ['raw-loader', 'less-loader'] },
        { test: /\.scss$|\.sass$/, loaders: ['raw-loader', 'sass-loader'] },
        { test: /\.(jpg|png)$/, loader: 'url-loader?limit=128000' }
      ]
    },

    resolve: {
      extensions: ['.js', '.ts'],
      modules: [
        path.join(wallaby.projectCacheDir, 'src/app'),
        path.join(wallaby.projectCacheDir, 'src'),
        'node_modules'
      ],
      alias: {
        // except for @angular, this should match tsconfig.spec.json
        '@environment': path.join(wallaby.projectCacheDir, 'src/environments'),
        '@assets': path.join(wallaby.projectCacheDir, 'src/assets'),
        "@utils": path.join(wallaby.projectCacheDir, 'src/app/utils'),
        "@shared": path.join(wallaby.projectCacheDir, 'src/app/shared'),
        "@invoiceModels": path.join(wallaby.projectCacheDir, '/src/app/features/invoices/models'),
        "@models": path.join(wallaby.projectCacheDir, 'src/app/models'),
        "@core": path.join(wallaby.projectCacheDir, 'src/app/core'),
      }
    },
    node: {
      fs: 'empty',
      net: 'empty',
      tls: 'empty',
      dns: 'empty'
    }
  });

  return {
    files: [
      { pattern: 'src/**/*.+(ts|css|less|scss|sass|styl|html|json|svg)', load: false },
      { pattern: 'src/**/*.d.ts', ignore: true },
      { pattern: 'src/**/*.spec.ts', ignore: true },
      { pattern: 'src/**/*.dev.ts', ignore: true }
    ],
    filesWithNoCoverageCalculated: [
      'src/**/*.mock.ts'
    ],

    tests: [
      { pattern: 'src/**/*spec.ts', load: false },
      { pattern: 'src/**/*e2e-spec.ts', ignore: true }
    ],

    testFramework: 'jasmine',

    compilers: {
      '**/*.ts': wallaby.compilers.typeScript(compilerOptions)
    },

    middleware: function (app, express) {
      var path = require('path');
      app.use('/favicon.ico', express.static(path.join(__dirname, 'src/favicon.ico')));
      app.use('/assets', express.static(path.join(__dirname, 'src/assets')));
    },

    env: {
      kind: 'chrome'
    },

    postprocessor: webpackPostprocessor,

    setup: function () {
      window.__moduleBundler.loadTests();
    },

    debug: true,
    lowCoverageThreshold: 70
  };
};
```

### Code editor or IDE name and version

Visual Studio Code v1.24 (May 2018)

### OS name and version

OSX 10.13.4 High Sierra

**Comment by ArtemGovorov** [comment]
> Cannot find module '1922' (undefined)​​

Given the error message, it's highly likely that one of your specs is importing/referencing another spec file. Please check if it is the case, and if so, try refactoring the reused bit into a file of its own.

**Comment by jkyoutsey** [comment]
@ArtemGovorov  I have searched all of my *.spec.ts files for any reference to a *.spec import.  None exist.  Other ideas?  The module number does seem to change.  Today it's 1655.

Is there any way to determine what spec files are at fault?  Or at least what chunk of them is being skipped?  That would narrow the scope.  This is a rather large project for random searching.

**Comment by jkyoutsey** [comment]
@ArtemGovorov PERFECT!  Exactly the right solution.  Someone had written a mock in one spec and re-used it from other specs.  Thank you!

**Comment by michael-letcher** [comment]
@ArtemGovorov would it be possible to add a slightly better error than a number? Even suggesting that

> [It is] highly likely that one of your specs is importing/referencing another spec file. Please check if it is the case, and if so, try refactoring the reused [code] into a file of its own.

**Comment by ArtemGovorov** [comment]
@michael-letcher After some investigation, we have added better reporting about the issue and published it in `wallaby-webpack` v3.9.13. Please update the package in your project to the latest version and the reporting should be much better.

**Comment by michael-letcher** [comment]
@ArtemGovorov Thankyou!
